frontend-service/src/test/java/com/amazonaws/blox/frontend/integration/StartDeploymentIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.frontend.integration;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.StartDeploymentResponse;
+import com.amazonaws.blox.frontend.operations.StartDeployment;
+import com.amazonaws.serverless.proxy.internal.model.AwsProxyRequest;
+import com.amazonaws.serverless.proxy.internal.model.AwsProxyResponse;
+import com.amazonaws.serverless.proxy.internal.testutils.AwsProxyRequestBuilder;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class StartDeploymentIntegrationTest extends IntegrationTestBase {
+
+  private static final String DEPLOYMENT_ID = "deployment-id";
+  private static final String ENVIRONMENT_REVISION_ID = "1";
+
+  @Test
+  public void testStartDeployment() throws Exception {
```

Comment:
Do we need to test any failure cases as well? Like when the response fails.

Reply:
yes, definitely, like one exception for ResourceNotFoundException.

Reply:
I added one failure case. However, we don't handle exception properly.  It's a big topic as we need to probably do Spring error handling or AWS lambda error handling, and may related with API and client SDK change.

Reply:
Yeah, there's still an open task for this in #280, which we should break out for post M1.